testutil/client.go

```diff
@@ -288,24 +291,28 @@ func HttpLogin(params *LoginParams) (*HttpToken, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "unable to marshal body")
 	}
+	fmt.Println("HttpLogin: body", body)
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein

note: need to rebase